### PR TITLE
strategies: Add authentication info in bound client

### PR DESCRIPTION
**Pull request by MarcoWorms**
Add client.authentication variable after binding the option to the client, this is usefull for getting your session info when authenticating with email/password for example.

### CODE REVIEW

File: lib/resources/index.spec.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import client from '../client'
```
Comment (derekstavis) [inline lib/resources/index.spec.js]: Why change this? Looks unrelated to this pull request.

Comment (derekstavis) [inline lib/resources/index.spec.js]: What about using `import pagarme from '../..'` ?

Comment (MarcoWorms) [inline lib/resources/index.spec.js]: done ;)
